deploy/staging/application/containers_definition.json

```diff
@@ -0,0 +1,163 @@
+        "name": "POSTGRES_HOST",
+        "valueFrom": "${POSTGRES_HOST}"
+      },
+      {
+        "name": "SECRET_KEY",
+        "valueFrom": "${SECRET_KEY}"
+      }
+    ]
+  },
+  {
+    "name": "${backend_container_name}",
+    "image": "${backend_image_url}@${backend_digest}",
+    "essential": true,
+    "dockerLabels": {
+      "Project": "SOLO"
+    },
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${ecs_task_def_service_name}_backend_logGroup",
+        "awslogs-region": "${region}",
+        "awslogs-stream-prefix": "${ecs_task_def_service_name}_backend_logGroup_stream"
+      }
+    },
+    "portMappings": [
+      {
+        "hostPort": 8000,
+        "protocol": "tcp",
+        "containerPort": 8000
+      }
+    ],
+    "dependsOn": [
+      {
+        "containerName": "${backend_container_name}-migrate",
+        "condition": "SUCCESS"
+      }
+    ],
+    "secrets": [
+      {
+        "name": "POSTGRES_DB",
+        "valueFrom": "${POSTGRES_DB}"
+      },
+      {
+        "name": "POSTGRES_USER",
+        "valueFrom": "${POSTGRES_USER}"
+      },
+      {
+        "name": "POSTGRES_PASSWORD",
+        "valueFrom": "${POSTGRES_PASSWORD}"
+      },
+      {
+        "name": "POSTGRES_HOST",
+        "valueFrom": "${POSTGRES_HOST}"
+      },
+      {
+        "name": "MAIN_DOMAIN",
+        "valueFrom": "${MAIN_DOMAIN}"
+      },
+      {
+        "name": "API_DOMAIN",
+        "valueFrom": "${API_DOMAIN}"
+      },
+      {
+        "name": "AUTH_DOMAIN",
+        "valueFrom": "${AUTH_DOMAIN}"
+      },
+      {
+        "name": "SECRET_KEY",
+        "valueFrom": "${SECRET_KEY}"
+      }
+    ]
+  },
+  {
+    "name": "${frontend_container_name}",
+    "image": "${frontend_image_url}@${frontend_digest}",
+    "essential": true,
+    "dockerLabels": {
+      "Project": "SOLO"
+    },
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${ecs_task_def_service_name}_frontend_logGroup",
+        "awslogs-region": "${region}",
+        "awslogs-stream-prefix": "${ecs_task_def_service_name}_frontend_logGroup_stream"
+      }
+    },
+    "secrets": [
+      {
+        "name": "MAIN_DOMAIN",
+        "valueFrom": "${MAIN_DOMAIN}"
+      },
+      {
+        "name": "NGINX_SSL_CERT",
+        "valueFrom": "${NGINX_SSL_CERT}"
+      },
+      {
+        "name": "NGINX_SSL_KEY",
+        "valueFrom": "${NGINX_SSL_KEY}"
+      },
+      {
+        "name": "API_DOMAIN",
+        "valueFrom": "${API_DOMAIN}"
+      },
+      {
+        "name": "AUTH_DOMAIN",
+        "valueFrom": "${AUTH_DOMAIN}"
+      },
+      {
+        "name": "BACKEND_PROXY",
+        "valueFrom": "${BACKEND_PROXY}"
+      }
+    ],
+    "portMappings": [
+      {
+        "hostPort": 443,
+        "protocol": "tcp",
+        "containerPort": 443
+      },
+      {
+        "hostPort": 80,
+        "protocol": "tcp",
+        "containerPort": 80
+      }
```

Comment:
Is there ever a chance of port 80 making it to our frontend container for this to apply?

Reply:
deleted. we don't need that object. it goes straight to 443